examples/model_compression/distill_lstm/data.py

```diff
@@ -249,7 +268,7 @@ def create_distill_loader(task_name,
         vocab = tokenizer
         pad_val = tokenizer.pad_token_id
 
-    if task_name == 'senta':
+    if task_name == 'chnsenticorp':
         train_ds = apply_data_augmentation_for_cn(
```

Comment:
Is it possible to use the `dataset.map()` to do augmentation? @smallv0221 Discuss it with @LiuChiachi

Reply:
yes

Reply:
Yes. Done. Thanks for your suggestions